src/controller/RentalsController.java

```diff
@@ -168,4 +179,46 @@ public static Rental getRental(int rid) {
 
         return retRental;
     }
+
+
+    public static Vehicle rentVehicle(int confNo, String cardName, int cardNo, String expDate){
+        Reservation reservation = ReservationsController.getReservation(confNo);
```

Comment:
Hmm I think we should get the reservation before calling "rentVehicle" and we should pass the reservationModel object into "rentVehicle" instead of confNo